README.md

```diff
@@ -17,6 +17,13 @@
 
 µTask allows you to model business processes in a **declarative yaml format**. Describe a set of inputs and a graph of actions and their inter-dependencies: µTask will asynchronously handle the execution of each action, working its way around transient errors and keeping a trace of all intermediary states until completion.
 
+Here are some example usecases for µTask:
+- trigger remediation commands across a farm of machines over ssh
```

Comment:
I think this example can be replaced by a more top-level example, because the first sight about the project will be "oh it's another kind of Ansible". WDYT?